src/main/resources/static/assets/js/sdh.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+	var app = new Vue({
```

Comment:
Do you actually need the `app` variable here? You're not using it anywhere. If you do need it, it should be `let` or `const` for consistency.

Reply:
No, this was copy/paste. I'll remove.